lua/nvim-treesitter/query.lua

```diff
@@ -105,7 +105,7 @@ do
   ---@param query_name string
   function M.get_query(lang, query_name)
     if cache[lang][query_name] == nil then
-      cache[lang][query_name] = ts.get_query(lang, query_name)
+      cache[lang][query_name] = ts.get(lang, query_name)
```

Comment:
I believe this is not the cause for the deprecation notice but somewhere else. Can you have a look at where the problem might be?